views/pages/index.ejs

```diff
@@ -2,6 +2,6 @@
 <% include ../partials/header %>
 </head>
 <div class="centered">
-	<h1 id="header">Technoramic 5190</h1>
-	<em>We are the co-ed high school team of MICDS, non-profit and independent school in Saint Louis, Missouri.</em>
+	<h1 id="header">Welcome to our website!</h1>
```

Comment:
maybe this seems kind of tacky? is there another thing we could put here? I like the idea that it's being changed though